library/ws-widget/collection/src/lib/content-strip-with-tabs/content-strip-with-tabs.component.ts

```diff
@@ -445,7 +451,35 @@ export class ContentStripWithTabsComponent extends WidgetBaseComponent
                             strip: NsContentStripWithTabs.IContentStripUnit) {
     const inprogress: any[] = []
     const completed: any[] = []
-    array.forEach((e: any, idx: number, arr: any[]) => (customFilter(e, idx, arr) ? inprogress : completed).push(e))
+    // array.forEach((e: any, idx: number, arr: any[]) => (customFilter(e, idx, arr) ? inprogress : completed).push(e))
+    array.forEach((e, idx, arr) => {
+      const statusRetired = (e.status as string).toLowerCase()
```

Comment:
first put condition for e.status , then do lowercase conversion

Reply:
changed